docs/sources/userguide/dockervolumes.md

```diff
@@ -132,6 +134,11 @@ And another:
 
     $ sudo docker run -d --volumes-from dbdata --name db2 training/postgres
 
+It is important to note that if the `postgres` image contained a directory
```

Comment:
In this case, if the `postgres` image contained a directory
called `/dbdata` then mounting the volumes from the `dbdata` container 
hide sthe `/dbdata` files from the `postgres` image. The result is only
the files from the `dbdata` container are visible.